app/src/test/kotlin/com/thundermaps/apilib/android/api_impl/resources/ReportImplTest.kt

```diff
@@ -32,6 +33,12 @@ class ReportImplTest {
+        val report = Gson().fromJson(response, Report::class.java)
+        assert(report.form_fields != null)
+    }
```

Comment:
To avoid redeclaring the long response json test string, can we use the one already defined as a class variable (reportTestpayload). So: 

```
val report = Gson().fromJson(reportTestpayload, Report::class.java)
assert(report.form_fields != null)
```